openwisp_controller/pki/migrations/__init__.py

```diff
@@ -1,13 +1,20 @@
+import swapper
 from django.contrib.auth.models import Permission
 
 from ...migrations import create_default_permissions
 
 
+def get_swapped_model(apps, app_name, model_name):
+    model_path = swapper.get_model_name(app_name, model_name)
+    app, model = swapper.split(model_path)
+    return apps.get_model(app, model)
```

Comment:
this function looks repeated for different apps, can you please make 1 function for all of them?